src/ResourceManager/Kusto/Kusto.sln

```diff
@@ -0,0 +1,169 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27004.2005
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.Kusto", "Commands.Kusto\Commands.Kusto.csproj", "{D2A8F744-8DCB-4A13-BA80-EB181FF365AD}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.Common", "..\..\Common\Commands.Common\Commands.Common.csproj", "{5EE72C53-1720-4309-B54B-5FB79703195F}"
```

Comment:
All of these common projects don't exist in this repo anymore. Please pull the latest from the preview branch.